apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -181,15 +187,30 @@ export abstract class BaseInstallAction extends BaseRushAction {
     success: boolean
   ): void {
     if (this.parser.telemetry) {
+      let extraData: { [key: string]: string } = {
+        mode: this.actionName,
+        clean: (!!this._purgeParameter.value).toString(),
+        bypassPolicy: (!!this._bypassPolicyParameter.value).toString(),
+        noLink: (!!this._noLinkParameter.value).toString(),
+        networkConcurrency: this._networkConcurrencyParameter.value
+          ? this._networkConcurrencyParameter.value.toString()
+          : 'unspecified',
+        debugPackageManager: (!!this._debugPackageManagerParameter.value).toString(),
+        maxInstallAttempts: this._maxInstallAttempts.value
+          ? this._maxInstallAttempts.value.toString()
+          : 'unspecified',
+        ignoreHooks: (!!this._ignoreHooksParameter.value).toString(),
+        debug: installManagerOptions.debug.toString(),
+        full: installManagerOptions.fullUpgrade.toString()
```

Comment:
Can we get this from the set of all parameters (like via the `this.parameters` object) instead of enumerating them explicitly?